desktop/release-notes.md

```diff
@@ -24,34 +24,108 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
+- Container File Explorer is available as an experimental feature - you now have the ability to debug the filesystem within your containers in the GUI.
+- Global Search now includes Volume results.
+
+### Upgrades
+
+- [Containerd v1.6.18](https://github.com/containerd/containerd/releases/tag/v1.6.18), which includes fixes for [CVE-2023-25153](https://github.com/advisories/GHSA-259w-8hf6-59c2) and [CVE-2023-25173](https://github.com/advisories/GHSA-hmfx-3pcx-653p).
+- [Docker Engine v20.10.23](https://docs.docker.com/engine/release-notes/20.10/#201023).
+- [Go 1.19.5](https://github.com/golang/go/releases/tag/go1.19.5)
+
+### Bug fixes and enhancements
+
+#### For all platforms
+
+- Fixed a bug where diagnostic gathering could hang waiting for a subprocess to exit.
+- Prevented the transparent HTTP proxy from mangling requests too much. Fixes tailscale extension login, see [tailscale/docker-extension#49](https://github.com/tailscale/docker-extension/issues/49).
+- Fixed a bug in the transparent TLS proxy where the Server Name Indication field is not set.
+- Added support for subdomain match, CIDR match, `.` and `*.` in HTTP proxy exclude lists.
+- Ensured HTTP proxy settings are respected when uploading diagnostics.
+- Fixed fatal error when fetching credentials from the credential helper.
+- Fixed fatal error related to concurrent logging.
+- Improved the UI for Extensions actions in the Marketplace.
+- Added a new filter in Extensions Marketplace which includes both Categories and Reviewed status.
+- Added a way to report to Docker an extension as malicious.
+- Updated Dev Environments to v0.2.2 with initial set up reliability & security fixes.
+- Added a whalecome survey for new users only.
+- Made the troubleshooting page's confirmation dialogs style consistent with other similar dialogs.
+- Fixed fatal error caused by resetting the Kubernetes cluster before it has started.
+- Implemented `docker import` for the containerd integration.
+- Fixed image tagging with an existing tag with the containerd integration.
+- Implemented the dangling filter on images for the containerd integration.
+- Fixed `docker ps` failing with containers whose images are no longer present with the containerd integration.
+
+#### For Mac
+
+- Fixed download of Registry Access Management policy on systems where the privileged helper tool `com.docker.vmnetd` is not installed.
+- Fixed a bug where `com.docker.vmnetd` could not be installed if `/Library/PrivilegedHelperTools` does not exist.
+- Fixed a bug where the "system" proxy would not handle "autoproxy" / "pac file" configurations.
+- Fixed a bug where vmnetd installation fails reading `Info.Plist` on case-sensitive file systems. Actual filename is `Info.plist`. Fixes [docker/for-mac#6677](https://github.com/docker/for-mac/issues/6677).
+- Fixed the start of the application on login not working.
```

Comment:
We need also a release note for a bug where user is always prompted on startup to create the docker socket symlink.
Maybe we could have something like this:
```
Fix a bug where user is prompted to create the docker socket symlink on every startup.
Fixes [for-mac#6634](https://github.com/docker/for-mac/issues/6634)
```